metricbeat/docs/modules/activemq.asciidoc

```diff
@@ -35,6 +35,36 @@ metricbeat.modules:
   path: '/api/jolokia/?ignoreErrors=true&canonicalNaming=false'
   username: admin # default username
   password: admin # default password
+  processors:
+    - script:
+        lang: javascript
+        source: >
+          function process(event) {
+            var broker_memory_broker_pct = event.Get("activemq.broker.memory.broker.pct")
+            if (broker_memory_broker_pct != null) {
+              event.Put("activemq.broker.memory.broker.pct", broker_memory_broker_pct / 100.0)
+            }
+
+            var broker_memory_temp_pct = event.Get("activemq.broker.memory.temp.pct")
+            if (broker_memory_temp_pct != null) {
+              event.Put("activemq.broker.memory.temp.pct", broker_memory_temp_pct / 100.0)
+            }
+
+            var broker_memory_store_pct = event.Get("activemq.broker.memory.store.pct")
+            if (broker_memory_store_pct != null) {
+              event.Put("activemq.broker.memory.store.pct", broker_memory_store_pct / 100.0)
+            }
+
+            var queue_memory_broker_pct = event.Get("activemq.queue.memory.broker.pct")
+            if (queue_memory_broker_pct != null) {
+              event.Put("activemq.queue.memory.broker.pct", queue_memory_broker_pct / 100.0)
+            }
+
+            var topic_memory_broker_pct = event.Get("activemq.topic.memory.broker.pct")
+            if (topic_memory_broker_pct != null) {
+              event.Put("activemq.topic.memory.broker.pct", topic_memory_broker_pct / 100.0)
+            }
+          }
```

Comment:
this is great! pinging @jsoriano we should probably support this in the metricset manifest, WDYT?

Reply:
Yes, I have added an issue for that #14730 

I wouldn't add this to the user configuration because we cannot update these files when they are in use.

Reply:
@mtojek @exekias I have added an issue so we don't forget to move the processors to `manifest.yml` when possible: https://github.com/elastic/beats/issues/14742

I wouldn't release any module as GA with processors in the configuration.

Reply:
I don't think #14730 is the right link there 😬 @jsoriano

Reply:
@kaiyan-sheng thanks! link fixed